docs/fundamentals/write-operations.txt

```diff
@@ -480,3 +479,139 @@ with ``"contemporary"`` in the ``genres`` array field. Click the
 
 To learn more about array update operators, see :manual:`Array Update Operators </reference/operator/update-array/>`
 in the {+server-docs-name+}.
+
+.. _laravel-fundamentals-delete-documents:
+
+Delete Documents
+----------------
+
+In this section, you can learn how to delete documents from a MongoDB collection
+by using {+odm-short+}. Use delete operations to remove data from your MongoDB
+database permanently.
```

Comment:
"permanently" is wrong when the `SoftDeletes` trait is used. But you mention it just below, so maybe that's okay.

Reply:
Thanks for providing that perspective, I'll update the text to make that more clear!